CRM/Iats/Form/Settings.php

```diff
@@ -139,7 +139,7 @@ public function postProcess() {
         unset($values[$key]);
       }
     }
-    CRM_Core_BAO_Setting::setItem($values, 'iATS Payments Extension', 'iats_settings');
+    Civi::settings()->set('iats_settings', $values);
     parent::postProcess();
```

Comment:
I thought I saw a `get` here - did you just fix that?

Reply:
Correct @KarinG :)